emscripten.py

```diff
@@ -737,7 +737,7 @@ def create_receiving(exports):
       for s in exports_that_are_not_initializers:
         mangled = asmjs_mangle(s)
         dynCallAssignment = ('dynCalls["' + s.replace('dynCall_', '') + '"] = ') if generate_dyncall_assignment and mangled.startswith('dynCall_') else ''
-        receiving += [dynCallAssignment + mangled + ' = asm["' + s + '"];']
+        receiving += [dynCallAssignment + mangled + ' = Module["{}"] = asm["{}"];'.format(mangled, s)]
```

Comment:
I think this should only be in he `MODULARIZE` case.

Reply:
Done 👌